syn_nom/src/lib.rs

```diff
@@ -10,26 +10,29 @@ pub enum IResult<I, O> {
     Error,
 }
 
```

Comment:
Please also move all the macros in src/helper.rs - you won't get very far without those, especially `punct!` and `keyword!`. Keep them in a separate file though to track that they are not from upstream nom.